src/views/homepage/components/featured-content-grid/search-featured-card.module.css

```diff
@@ -0,0 +1,122 @@
+.searchCard {
+	background: url('../../img/search-card-graphic.svg');
```

Comment:
💡 perhaps each card should have its own dir within components and the image dir for this could live there?